cloud/pkg/controller/messagelayer/util.go

```diff
@@ -34,26 +44,71 @@ func GetNodeID(msg model.Message) (string, error) {
 // GetNamespace from "beehive/pkg/core/model".Model.Router.Resource
 func GetNamespace(msg model.Message) (string, error) {
 	sli := strings.Split(msg.GetResource(), constants.ResourceSep)
-	if len(sli) <= controller.ResourceNamespaceIndex {
+	length := controller.ResourceNamespaceIndex
+	if config.EdgeSiteEnabled {
+		length = controller.EdgeSiteResourceNamespaceIndex
+	}
+	if len(sli) <= length {
 		return "", fmt.Errorf("namespace not found")
 	}
-	return sli[controller.ResourceNamespaceIndex], nil
+	var res string
+	var index uint8
+	if config.EdgeSiteEnabled {
+		res = sli[controller.EdgeSiteResourceNamespaceIndex]
+		index = controller.EdgeSiteResourceNamespaceIndex
+	} else {
+		res = sli[controller.ResourceNamespaceIndex]
+		index = controller.ResourceNamespaceIndex
+	}
+	log.LOGGER.Infof("The namespace is %s, %d", res, index)
```

Comment:
do we need this log ? can it be at debug level ? also the format of message can be improved to indicate the index.

Reply:
Done. Thanks